File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int>st;
        for(auto num : nums)
        {
            st.insert(num);

        }
        int maxlen =0;
        for(int i=0;i<nums.size();i++)
        {
            int count = 1;
            int num = nums[i];
            while(true)
            {
                if(st.count(num+1)) {count++;num +=1;}
                else break;
            }
            maxlen = max(maxlen,count);
        }
        return maxlen;
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        n = len(nums)
        num = 0
        count = 0
        for i in range(0,n):
            if count == 0:
                num = nums[i]
                count+=1
            else:
                if nums[i]!=num:
                    count-=1
                else:
                    count+=1
        return num